import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,24 @@
+import {S3CreateEvent} from 'aws-lambda';
+import {S3Client} from '@aws-sdk/client-s3';
+import * as process from 'process';
+import middy from '@middy/core';
+import * as console from 'console';
+import {copyParsedFile, deleteParsedFile, executeFileParse} from './utils';
+
+const importFileParser = async (event: S3CreateEvent) => {
+    console.log('importFileParser event: ', event);
+
+    const key = decodeURIComponent(event.Records[0].s3.object.key.replace(/\+/g, ' '));
+    const s3Client = new S3Client({region: process.env.REGION});
```

Comment:
Im not sure, but you can create s3Client on tol lvl only once, also, dont think passing it to async functions each time is nice, better to use already existing one or create in functions by itself.
I think creating on top lvl is the best one. But I might be wrong ofc :D